packages/teams-js/src/public/marketplace.ts

```diff
@@ -0,0 +1,211 @@
+import { sendAndHandleSdkError } from '../internal/communication';
+import { ensureInitialized } from '../internal/internalAPIs';
+import { Currency, Locale, Market } from '.';
+import { errorNotSupportedOnPlatform, FrameContexts } from './constants';
+import { runtime } from './runtime';
+
+/**
+ * Represents the cart object for the app checkout flow.
+ *
+ * @beta
+ */
+export interface Cart {
+  /** Version of the cart. */
+  readonly version: CartVersion;
+  /** The id of the cart. */
+  readonly id: string;
+  /** The cart info. */
+  cartInfo: CartInfo;
+  /** The cart items. */
+  cartItems: CartItem[];
+}
+
+/**
+ * Version of the cart.
+ *
+ * @beta
+ */
+interface CartVersion {
+  /** Represents the major version number. */
+  majorVersion: number;
+  /** Represents the minor version number. */
+  minorVersion: number;
+}
+
+/**
+ * Represents the cart information
+ *
+ * @beta
+ */
+interface CartInfo {
+  /** The country market where the products are selling. */
+  readonly market: Market;
+  /** The identifier to tell the cart is checked out by admin or information worker */
+  readonly intent: Intent;
+  /** Locale for the user */
+  readonly locale: Locale;
+  /** The status of the cart. */
+  status: CartStatus;
+  /** Currency code for the cart item price. */
+  readonly currency: Currency;
+  /** ISO 8601 timestamp string, indicates when the cart is created. */
+  readonly createdAt: string;
+  /** ISO 8601 timestamp string, indicates when the cart is updated. */
+  updatedAt: string;
+}
+
+/**
+ * Represents the basic cart item information.
+ *
+ * @beta
+ */
+interface Item {
+  /** The id of the cart item. */
+  readonly id: string;
+  /** The display name of the cart item. */
+  name: string;
+  /** The quantity of the cart item. */
+  quantity: number;
+  /** The price of the single cart item. */
+  price: number;
+  /** The thumbnail imageURL of the cart item. */
+  readonly imageURL?: string;
+}
+
+/**
+ * Represents the cart item that could have accessories
+ *
+ * @beta
+ */
+export interface CartItem extends Item {
+  /** Accessories to the item if existing. */
+  readonly accessories?: Item[];
+}
+
+/**
+ * Represents the persona creating the cart.
+ *
+ * @beta
+ */
+export enum Intent {
+  /** The cart is created by admin of an organization. */
+  AdminUser = 'AdminUser',
+  /** The cart is created by end user of an organization. */
+  EndUser = 'EndUser',
+}
+
+/**
+ * Represents the status of the cart.
+ *
+ * @beta
+ */
+export enum CartStatus {
+  /** Cart is created but not checked out yet. */
+  Open = 'Open',
+  /** Cart is checked out but not completed yet. */
+  Processing = 'Processing',
+  /**
+   * Indicate checking out is completed and the host should
```

Comment:
I don't think I know what this sentence means. Is the end of the sentence missing?